kubernetes/start/intel/adoc/gs_rancher_intel-flexran.adoc

```diff
@@ -46,7 +46,7 @@ include::./common_docinfo_vars.adoc[]
 :title: Intel® FlexRAN™ - SUSE Reference Solution Cloud-Native Setup
 :subtitle: FlexRAN™ Deployment Guide on SUSE platform
 :productname:
-//  SLES 15 SP3 RT, SLE Micro 5.2 RT, RANCHER 2.6, RKE2
+// It's not about a specific product but rather a platform with multiple SUSE productsinlcuging SLES 15 SP3 RT, SLE Micro 5.2 RT, RANCHER 2.6, RKE2
```

Comment:
Guides are organized according to product focus.  While multiple products/platforms can be included, the product/platform highest in the stack is typically designated for organizational purposes.   The 'productname' variable is used to designate this.